config/repo-values.yaml

```diff
@@ -222,3 +222,5 @@ sync:
       url: https://concourse-charts.storage.googleapis.com/
     - name: tomkerkhove
       url: https://tomkerkhove.azurecr.io/helm/v1/repo
+    - name: jenkins
```

Comment:
@tomaszsek The name in both files needs to be the same. Can you update them to both be the same.